Extension/Blazor-Extension/Visual-Studio-Code/create-project.md

```diff
@@ -17,11 +17,11 @@ The instructions below assist you in creating **Syncfusion Blazor Applications**
 
 1. To create a Syncfusion Blazor application in Visual Studio Code, open the command palette by pressing **Ctrl+Shift+P**. Search for the word **Syncfusion** in the Visual Studio Code palette to get the templates provided by Syncfusion.
 
-    ![CreateProjectPalette](images/CreateBlazorProjectPalette.png)
+    ![Create Project Palette in Blazor Extension](create-project_images/blazor-extension-project-palette.png)
```

Comment:
Seems image folder name is wrong. Why you change the image folder name?
Previously they used images folder name, you changed to create-project_images. It is correct?
Don't change the image folder name. If the folder name has capital letter, you should change to small casing. Otherwise don't change the folder name.

Check and update the changes in all your PR's.

Reply:
Made changes as per the comment